src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -381,6 +382,7 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
           parentColumns={gridColumns}
         >
           <main role="main">
+            <Text>Hellooooo I am TypeScript!</Text>
```

Comment:
Does this pass the CI checks?  I thought `className` would need to be passed as it's not marked as optional in `TextProps`.

Reply:
I think because this is jsx that it's not picked up here

Reply:
Yeh it is. When I use that component in a tsx file then I get the errors

![Screenshot 2022-07-28 at 13 11 37](https://user-images.githubusercontent.com/4798332/181501970-f854c5d1-c938-465b-b688-dc379edc8ab5.png)

Reply:
Aah of course - cheers 👍